src/vs/workbench/browser/parts/menubar/menubarPart.ts

```diff
@@ -438,10 +439,7 @@ export class MenubarPart extends Part {
 	}
 
 	private setupNativeMenubar(): void {
-		// TODO@sbatten: Remove once native menubar is ready
-		if (isMacintosh && isWindows) {
-			this.menubarService.updateMenubar(this.windowService.getCurrentWindowId(), this.getMenubarMenus());
-		}
+		this.menubarService.updateMenubar(this.windowService.getCurrentWindowId(), this.getMenubarMenus());
```

Comment:
I do not really see a point of a private method called just once that has a one line call. Just my 2 cents from the birds eye perspective

Reply:
The reason is primarily for symmetry. Where this is called we call either custom or native and the custom one is super long. I could make less functions and more if statements to achieve the same level of readability or just add a comment and remove the private function

Reply:
Whatever you prefer :)